app/models/country_petition_journal.rb

```diff
@@ -40,6 +42,7 @@ def self.reset!
         timezone('utc', now()), timezone('utc', now())
       FROM signatures
       WHERE state = 'validated'
+      AND country <> ''
```

Comment:
Hmm, appears to be true when I just tested it - can you rebase and then I'll merge it, thanks.